src/verbs.js

```diff
@@ -0,0 +1,28 @@
+import { Request } from './request'
```

Comment:
The class name is `FetchRequest` instead of `Request`. We've added a prefix to the class names to avoid clashes with built-in classes as pointed here https://github.com/rails/request.js/issues/3

Reply:
it is corrected. thanks!